File: tmp-mei/fgenjs5030319945734616766/base/Todo_0.js
import * as rt from "../rt/main.js";
export class base$$Todo_0 {
  static $self = new base$$Todo_0();

  $exclamation$imm() {
  return base$$Todo_0.$exclamation$imm$fun(this);
}


$exclamation$imm(msg_m$) {
  return base$$Todo_0.$exclamation$imm$fun(msg_m$, this);
}

  static $exclamation$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static $exclamation$imm$fun(msg_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/Todo_0.java
package base;
public interface Todo_0{
Todo_0 $self = new Todo_0Impl();
Object $exclamation$imm();

Object $exclamation$imm(rt.Str msg_m$);
static Object $exclamation$imm$fun(base.Todo_0 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}

static Object $exclamation$imm$fun(rt.Str msg_m$, base.Todo_0 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}
}